### ISSUE TITLE
Error  java.lang.NullPointerException when loading individual variable set

### ISSUE THREAD

**Issue by pilarnatividad** [opencb/opencga#1390]
Hi, 
We want to use the Variable Sets and Annotation Sets used in the examples of the Annotation and Clinical Data section, and we are getting the **error java.lang.NullPointerException**  when executing the following command line:

 .`/opencga.sh variables create --json /home/opencga/individual_vs.json -n 'individual_private_details'  --description "Private details of the individual" -s 'pnatividad@reference_grch37:1kG_phase3' --of JSON --session-id 'eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJwbmF0aXZpZGFkIiwiYXVkIjoiT3BlbkNHQSB1c2VycyIsImlhdCI6MTU3MTQxNDg1MiwiZXhwIjoxNTcxNDE4NDUyfQ.Vm4W40WFETe_xZozrs3D4u3sVZoZrdlwwvnubzbcc7w' --log-level 'trace'`

```2019-10-18 18:09:09 [main] DEBUG CommandExecutor:137 - CLI session file is: CliSession{host='http://localhost:8080/opencga-1.4.1', version='v1', user='pnatividad', token='eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJwbmF0aXZpZGFkIiwiYXVkIjoiT3BlbkNHQSB1c2VycyIsImlhdCI6MTU3MTMyNjczMywiZXhwIjoxNTcxMzMwMzMzfQ.l1MRCiPKB9vaKPKIVoL51gblPoZYwJheCGTp_HNbUs4', login='20191017172055', expirationTime='20191017182055', projectsAndStudies={reference_grch37=[1kG_phase3], proyecto_incliva_1=[studio1_proyecto_incliva_1]}}
2019-10-18 18:09:09 [main] DEBUG VariableCommandExecutor:88 - sessionFile = CliSession{host='http://localhost:8080/opencga-1.4.1', version='v1', user='pnatividad', token='eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJwbmF0aXZpZGFkIiwiYXVkIjoiT3BlbkNHQSB1c2VycyIsImlhdCI6MTU3MTMyNjczMywiZXhwIjoxNTcxMzMwMzMzfQ.l1MRCiPKB9vaKPKIVoL51gblPoZYwJheCGTp_HNbUs4', login='20191017172055', expirationTime='20191017182055', projectsAndStudies={reference_grch37=[1kG_phase3], proyecto_incliva_1=[studio1_proyecto_incliva_1]}}
2019-10-18 18:09:09 [main] DEBUG VariableCommandExecutor:50 - Executing variables command line
2019-10-18 18:09:09 [main] DEBUG VariableCommandExecutor:85 - Creating variable
2019-10-18 18:09:09 [main] DEBUG VariableSetClient:243 - POST URL: http://localhost:8080/opencga-1.4.1/webservices/rest/v1/variableset/create?study=pnatividad%40reference_grch37%3A1kG_phase3&skip=0&limit=200&timeout=30000
2019-10-18 18:09:09 [main] DEBUG VariableSetClient:245 - Body org.opencb.commons.datastore.core.ObjectMap@e5027d45
```

The user pnatividad allows to the group admins.
What is the problem?
Thank you.

Pilar Natividad

**Comment by pfurio** [comment]
Hi Pilar, may I know which OpenCGA version are you running?

**Comment by pilarnatividad** [comment]
Hi Pedro, I running OpenCGA 1.4.1

Thank you

**Comment by pfurio** [comment]
Hi Pilar,

I just had a look at your issue and there are actually a few things going on. 
1. The examples from the documentation page were not 100% accurate and I just updated them. Basically, the individual variable set json contained the individual data model and the variable set instead of just the variable set. You can try with the new data again.
2. The command line described in the tutorial to create an annotation set has a bug and will fail if you try to run it once you successfully create the variable set. I will fix that in the hotfix-1.4.x branch in case you want to try it. We are expecting to create a new 1.4.2 release soon that will contain this fix as well.

However, you must know that the web services these command lines call and therefore, also the command lines you are using are deprecated since version 1.4.0 and will not work in next version 2.0.0. In fact, although we have created new web services to support these operations (variable set, annotation set creation/update) in a different way, they are actually not supported via our command line for being too complex (json files). For these situations, we expect users to use our Python or R client.

**Comment by pilarnatividad** [comment]
Hi Pedro,
Thank you for your clarification.
I have already installed  OpenCGA R client.

Pilar.
